app/assets/stylesheets/hubstats/application.css

```diff
@@ -12,9 +12,20 @@
  *= require hubstats/bootstrap
  *= require select2
  *= require select2-bootstrap
+ *= require bootstrap-datepicker
  *= require_tree .
  */
 
+.btn-default {
```

Comment:
Target a new class for this or make this one more specific.  `btn-default` is or could be used elsewhere since its a common bootstrap class.